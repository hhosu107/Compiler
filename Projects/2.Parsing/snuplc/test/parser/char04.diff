2c2
< parse error : at 18:1 : not statement
---
> parse error : at 18:1 : statement expected.
