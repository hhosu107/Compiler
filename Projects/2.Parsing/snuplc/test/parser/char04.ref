parsing 'char04.mod'...
parse error : at 18:1 : statement expected.


Done.
