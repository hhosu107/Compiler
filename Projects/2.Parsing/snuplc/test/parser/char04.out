parsing 'char04.mod'...
parse error : at 18:1 : not statement


Done.
